packages/manager/cypress/e2e/core/parentChild/token-expiration.spec.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  mockAppendFeatureFlags,
+  mockGetFeatureFlagClientstream,
+} from 'support/intercepts/feature-flags';
+import { mockGetLinodes } from 'support/intercepts/linodes';
+import { makeFeatureFlagData } from 'support/util/feature-flags';
+import {
+  accountFactory,
+  accountUserFactory,
+  profileFactory,
+} from 'src/factories';
+import { randomLabel, randomString } from 'support/util/random';
+import {
+  mockGetAccount,
+  mockGetChildAccounts,
+} from 'support/intercepts/account';
+import { mockGetProfile } from 'support/intercepts/profile';
+import { DateTime } from 'luxon';
+import { ui } from 'support/ui';
+
+const mockChildAccount = accountFactory.build({
+  company: 'Partner Company',
+});
+
+const mockChildAccountProxyUser = accountUserFactory.build({
+  username: randomLabel(),
+  user_type: 'proxy',
+});
+
+const mockChildAccountProfile = profileFactory.build({
+  username: mockChildAccountProxyUser.username,
+  user_type: 'proxy',
+});
+
+describe('Parent/Child token expiration', () => {
+  // @TODO M3-7554, M3-7559: Remove feature flag mocks after launch and clean-up.
+  beforeEach(() => {
+    mockAppendFeatureFlags({
+      parentChildAccountAccess: makeFeatureFlagData(true),
+    });
+    mockGetFeatureFlagClientstream();
+  });
+
+  /*
+   * - Confirms flow when a Proxy user attempts to switch back to a Parent account with expired auth token.
+   * - Uses mock API and local storage data.
+   */
+  it('shows session expiry prompt upon switching back to Parent account', () => {
```

Comment:
```suggestion
  it('shows session expiry prompt upon switching back to Parent account with expired Parent token', () => {
```
Small nitpick suggestion for clarity, just thinking about the test to follow in the future when the *proxy* token expires.

Reply:
Absolutely, these are both much clearer! Thanks @mjac0bs!

Reply:
[L51](https://github.com/linode/manager/pull/10341/files#diff-a4411cf9e5c7abfd273b78dd5c4b06281d6032972bd82182ed26d771b246bfd8R51) too 😅

Reply:
@mjac0bs Ah that's my bad, I should've realized that 🤦‍♂️ will fix tomorrow! Thanks again!